src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -37,31 +38,34 @@ public static enum GamePieceType {
 
 		}
 
-		// public final cone;
-		// public final cube;
-		// public final nothing;
+		// public final distance;
 
 	}
+	// HARDWARE
 
-	private final CANSparkMax motor;
+	private final CANSparkMax motor1;
+	private final CANSparkMax motor2;
 	// private final ColorSensorV3 colorSensor;
-	// private final distanceSensorIDK distanceSensor;
+	private final AnalogInput distanceSensor;
 
 	// CONSTRUCTOR
 	public IntakeSubsystem() {
-		motor = new CANSparkMax(INTAKE_MOTOR_1, MotorType.kBrushless);
+		motor1 = new CANSparkMax(INTAKE_MOTOR_1, MotorType.kBrushless);
+		motor2 = new CANSparkMax(INTAKE_MOTOR_2, MotorType.kBrushless);
 		// colorSensor = new ColorSensorV3(Port.kOnboard); //to find I2C port
-		// distanceSensor = new ();
-		motor.setIdleMode(IdleMode.kBrake);
+		motor1.setIdleMode(IdleMode.kBrake);
+		motor2.setIdleMode(IdleMode.kBrake);
+		distanceSensor = new AnalogInput(INTAKE_DISTANCE_SENSOR);
 	}
 
 	// METHODS
 	public void setSpeed(double speed) {
-		motor.set(speed);
+		motor1.set(speed);
+		motor2.set(-speed);
```

Comment:
If you find the need to start using motor commands in more methods, consider setting this motor to be inverted which can help make sure the sign is not forgotten.
Another option would be to setup motor2 to follow motor1.